src/js/mediaelementplayer-feature-ads.js

```diff
@@ -87,21 +87,29 @@
 
 			// add layer for ad links and skipping
 			player.adsLayer =
-					$('<div class="mejs-layer mejs-overlay mejs-ads">' +
+					$('<div class="' + t.options.classPrefix + 'layer ' +
+					                   t.options.classPrefix + 'overlay ' +
+									   t.options.classPrefix + 'ads">' +
```

Comment:
Should there maybe a helper method to build the class(es)? Similar to `mejs.i18n.t()`.